sqlmesh/cli/main.py

```diff
@@ -440,7 +440,14 @@ def ui(
     host = host or "127.0.0.1"
     port = 8000 if port is None else port
     os.environ["PROJECT_PATH"] = str(obj.path)
-    uvicorn.run("web.server.main:app", host=host, port=port, log_level="info")
+    uvicorn.run(
+        "web.server.main:app",
+        host=host,
+        port=port,
+        log_level="info",
+        timeout_graceful_shutdown=1,
```

Comment:
Is this needed? It was added to force the dev server to reload on code changes because it would wait indefinitely for SSE connections to terminate. The CLI disables reloading.